site/docs/foundations/zindex.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Z-Index
+layout: DetailTechnical
+sidebar:
+  label: Z-Index
+data:
+  status: In progress
+---
+
+Z-index determines the stacking order of elements. By providing consistent z-indexes, the design system ensures a predictable hierarchy of visual elements when stacked on top of one another. 
+
+A component with a greater z-index, or stack order, will always appear in front of an element with a lower stack order.
+
+Z-index is often used in combination with [Shadow](shadow) to help distinguish each layer in the stack.
+
+## Z-Index tokens
```

Comment:
I'm a bit worried that we have a lot of z-index tokens here that are quite specific to UI elements

Reply:
I agree. Lets discuss this in tomorrows call.